client/src/components/Profile/Profile.jsx

```diff
@@ -27,31 +126,185 @@ class Profile extends Component {
+              )}
+              {!isAuthenticated() && (
+                <Button onClick={this.login.bind(this)} color="inherit">
+                  Login
+                </Button>
+              )}
+
+              <IconButton
+                className={classes.menuItem}
+                color="inherit"
+                aria-label="Menu"
+                aria-owns={anchorEl ? "simple-menu" : undefined}
+                aria-haspopup="true"
+                onClick={this.handleClick}
+              >
+                <MenuIcon />
+              </IconButton>
+              <Menu
+                id="simple-menu"
+                anchorEl={anchorEl}
+                open={Boolean(anchorEl)}
+                onClose={this.handleClose}
+              >
+                {isAuthenticated() && (
+                  <MenuItem
+                    onClick={this.handleClose}
+                    className={classes.menuItem}
+                  >
+                    <ListItemIcon className={classes.icon}>
+                      <Person />
+                    </ListItemIcon>
+
+                    <ListItemText
+                      classes={{ primary: classes.primary }}
+                      inset
+                      primary="Profile"
+                      onClick={() => this.handleProfile()}
+                    />
+                  </MenuItem>
+                )}
+
+                <MenuItem
+                  onClick={this.handleClose}
+                  className={classes.menuItem}
+                >
+                  <ListItemIcon className={classes.icon}>
+                    <AccountBalance />
+                  </ListItemIcon>
+                  <ListItemText
+                    classes={{ primary: classes.primary }}
+                    inset
+                    primary="Home"
+                    onclick
+                  />
+                </MenuItem>
+                <MenuItem
+                  onClick={this.handleClose}
+                  className={classes.menuItem}
+                >
+                  <ListItemIcon className={classes.icon}>
+                    <LocationOn />
+                  </ListItemIcon>
+                  <ListItemText
+                    classes={{ primary: classes.primary }}
+                    inset
+                    primary="Location"
+                  />
+                </MenuItem>
+                <MenuItem
+                  onClick={this.handleClose}
+                  className={classes.menuItem}
+                >
+                  <ListItemIcon className={classes.icon}>
+                    <Fingerprint />
+                  </ListItemIcon>
+                  <ListItemText
+                    classes={{ primary: classes.primary }}
+                    inset
+                    primary="Logout"
+                  />
+                </MenuItem>
+              </Menu>
+            </Toolbar>
+          </AppBar>
+          <Paper className="profileBackground" elevation={20}>
```

Comment:
This profile code is brand new, but it looks really good! Not a ton goin on with this page, which is totally fine!